fm/views/user.py

```diff
@@ -29,8 +29,9 @@
 from fm.thirdparty.spotify import (
     PlaylistSerializer,
     SpotifyApi,
-    TrackSerializer
+    TrackSerializer,
 )
+from fm.serializers.spotify import TrackSerializer as FMTrackSerializer
```

Comment:
One reads from the Spotify REST API, the other reads from our DB. Our the serializer for the DB instances is in `fm.serializers.spotify` and the Spotify REST API is in `fm.thirdpart.spotify` which I think you added to read data from spotify's API.

The views `fm.views.user` for returning user stats was doing DB I/O but not using the correct serializer for that task which is why it was returning data the FE didn't expect so I swapped them out and the view will now return the correct data.

The naming and module location here is a bit off and does require more thought in the long run and really this code base needs a refactor to bring it up to our API spec, but thats another PR for another day :wink: